message-me/db/migrate/20210721103106_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :messages do |t|
+      t.text :body
```

Comment:
Why `text` ?

Reply:
should be text because it can hold more characters compared to string. Which is more suitable in our case.

Reply:
How much more characters?

Reply:
string is at 255 characters, while text is at 4294967296 default at 65536. which is more than enough.